product/reports/780_Tenants - Tenant Quotas/001_Tenant Quotas.yaml

```diff
@@ -6,6 +6,11 @@ rpt_group: Custom
 priority: 231
 rpt_type: Custom
 db: Tenant
+conditions: !ruby/object:MiqExpression
```

Comment:
Thanks @imtayadeway, I see the the expression does not support building native SQL for expressions containing the "count" operator. There's a TODO there from 2010! Nonetheless, there shouldn't be so many tenants and quotas that would cause the expression to run slowly.